README.md

```diff
@@ -1,26 +1,102 @@
+
+#### Map Styles
+[Map styles are specified in CartoCSS, the specification for which can be found here.](https://cartocss.readthedocs.io/en/latest/) The default stylesheet is located at `src/tiler/src/config/style.mss`. Multiple stylesheets can be used by adding them to the parameter “Stylesheet” in `src/tiler/src/config/map-config.mml`. Each .mss “class” refers to a map layer (its “id” property), specified in `map-config.mml`. 
+
+#### Filters
+Filters are specified by altering the query of one of your map’s layers in `src/tiler/src/config/style.mss`, and can then be fetched through the API. To create a filtered layer, modify the “table” property of the layer to have the format `(QUERY) as VARIABLE`. 
+   * Make sure each layer has unique “id” and “name” values.
+   * Make sure to include the geom column in this query, along with whatever columns you are filtering by.
+   * *Example:* `(SELECT geom,homes FROM table_name WHERE homes=’big’ ) as q` would create a filtered layer for all points that have a “homes” value of “big” in your database.
+   
+Layers can be accessed from the API by adding `?layers=layer1,layer2,etc.` as a query string to a tile or UTF grid URL. If no layers are specified in the query string, all layers specified in`map-config.mml` are displayed simultaneously.
+
+
+#### UTF Grids
+UTF grids can be generated at the url `/grid/{z}/{x}/{y}?utfFields=field1,field2,etc.`, where “utfFields” is a comma-separated list of one or more table columns from the database. Each grid url MUST have a “utfFields” query string. If you have already modified the table query of a layer to specify a filter, be sure the include the utf field column in the query. *Ex.* `(SELECT geom,homes,dog... )`, if you want to base a UTF grid off the value of “dog”.
+
+#### Vector Tiles
+Vector tiles of your data can be generated at the endpoint `/vector/{z}/{x}/{y}`. Filtering by layer works via query string but can also be handled client side, as can styling. [See here for more info about the Mapbox Vector Tile specification.](https://www.mapbox.com/vector-tiles/)
+
+#### Debugging
+The local development server exposes a Node.js process WebSocket that can be attached to your IDE of choice to step through and debug your code. Here are instructions on how to do so using Google Chrome's Dev Tools:
+ * Start the development server with `./scripts/server`.
+ * Open Google Chrome and navigate to `about:inspect`.
+ * There should be an option listed as something along the lines of "Target (v8.10.0)" with the Node.js logo and a path like `file:///home/tiler/bin`. Click "inspect", underneath.
+ * A new window will open up. Enter `ctrl+p` to open a search bar that lets you navigate to your source code. _The transpiled code in `src/tiler/bin/` is what is actually being tracked by the debugger,_  not the source code in `src/tiler/src/`.
+
+### Deployment to AWS
```

Comment:
I think a bit of intro would be helpful here before jumping into the steps.  I was thinking something like "A deployed Tilegarden endpoint consists of 1) a Lambda function to generate tiles, 2) an API Gateway endpoint to interpret URL parameters and route to the correct entrypoint in the Lambda function and 3) a CloudFront distribution to work around a limitation in API Gateway's image handling."

I see that you've already got a component rundown like this below at the `./scripts/publish` step, but I think it would be better to put it up front and have the `publish` section refer to it.